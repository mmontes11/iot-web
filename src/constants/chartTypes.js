export const BARCHART = "BARCHART";
export const LINECHART = "LINECHART";
