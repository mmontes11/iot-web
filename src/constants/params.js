export const FIRST_PARAM = "FIRST_PARAM";
export const SECOND_PARAM = "SECOND_PARAM";
