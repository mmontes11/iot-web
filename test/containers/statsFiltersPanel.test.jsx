import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import thunk from "redux-thunk";
import { MemoryRouter } from "react-router-dom";
import StatsFiltersPanel from "containers/statsFiltersPanel";
import { initialState } from "../constants";

describe("containers/statFiltersPanel", () => {
  it("renders in in initial state", () => {
    const store = configureStore([thunk])(initialState);
    const wrapper = shallow(
      <MemoryRouter initialEntries={["/"]} keyLength={0}>
        <Provider store={store}>
          <StatsFiltersPanel onFiltersChange={() => undefined} />
        </Provider>
      </MemoryRouter>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
