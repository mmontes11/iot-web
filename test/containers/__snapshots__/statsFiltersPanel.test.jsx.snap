// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/statFiltersPanel renders in in initial state 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "canGo": [Function],
      "createHref": [Function],
      "entries": Array [
        Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
      ],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "index": 0,
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Provider
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <withRouter(Connect(Connect(ResetOnUnmount)))
      onFiltersChange={[Function]}
    />
  </Provider>
</Router>
`;
